staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2894,6 +2894,9 @@ const (
 	// Tap domain attachment type is a generic way to bind ethernet connection into guests using tap device
 	// https://libvirt.org/formatdomain.html#generic-ethernet-connection.
 	Tap DomainAttachmentType = "tap"
+	// ManagedTap domain attachment type is binding an ethernet connection into guests using a tap device.
```

Comment:
Should we mention it is supported since 1.4 here as well?

Reply:
I guess we could move it to here (not duplicate).
But I do not think we have any policy about this at the moment.

Lest wait for an approver of this API area to feedback on this.